File: Notifications/Message.cs
﻿namespace Landlords.Notifications
{
    using System;

    public enum MessageType
    {
        Text,
        ClientMethodInvocation,
        ConnectionEvent
    }

    public class Message
    {
        public MessageType MessageType { get; set; }
        public string Data { get; set; }
    }
}

File: wwwroot/src/notifications/message.js
export let MessageType;
(function (MessageType) {
  MessageType[MessageType['Text'] = 0] = 'Text'
  MessageType[MessageType['MethodInvocation'] = 1] = 'MethodInvocation'
  MessageType[MessageType['ConnectionEvent'] = 2] = 'ConnectionEvent'
})(MessageType || (MessageType = {}))
export class Message {}
